### PR TITLE
Fix Spinner Loading Issue for Edge, IE11 and 10

### PR DESCRIPTION
**Pull request by nvasileiadis**
Closes #480

### CODE REVIEW

**Comment by jamesjacobs** [comment]
To confirm, I had this issue in Edge, IE11 and IE10.

**Comment by nvasileiadis** [comment]
Testing on everything mentioned and I believe it is in working condition now.

**Comment by jamesjacobs** [comment]
Changing the title confused me - so your changes fix Edge, IE11 and 10?

**Comment by nvasileiadis** [comment]
Yes all the usual browsers were tested. Please checkout this branch "480_ie10-spinner". The issue was quoted "spin". Now it should be ok.

File: stylesheets/_component.loading.scss [hunk 1]
```diff
@@ -96,6 +96,12 @@
         border: 0;
         text-indent: 0;
     }
+
+    .lt-ie10 & {
+        margin: 0 0 -2px;
+        background: url('../images/ie-loading.gif') center center no-repeat;
```
Comment (Stanton) [inline stylesheets/_component.loading.scss]: Good improvement, but could you use https://github.com/jadu/pulsar/blob/develop/images/loading.gif instead to match the non-gif one?

**Pull request merged**